randovania/games/fusion/logic_database/Sector 3 PYR.txt

```diff
@@ -330,13 +330,18 @@ Extra - room_id: [8]
 > Below E-Tank; Heals? False
   * Layers: default
   > Pickup (Hidden Energy Tank)
-      All of the following:
-          Can Break Single Bomb Blocks
-          Any of the following:
-              # TODO: space jump requires mid air morph if you have bombs. this whole block should probably get refacted a bit
-              Space Jump or Can Activate Pillar
-              # TODO: how is this supposed to be done if you have screw?
-              High Jump and Wall Jump (Beginner)
+      Any of the following:
+          All of the following:
+              # Use Pillar to Morph Ball Jump and place a Bomb
+              Can Use Bombs
+              # Bomb Jump is harder
+              Movement (Beginner) or Can Use Springball
+              # Pillar is hidden information, most players may not even know it exists unless you Power Bomb the room
+              Knowledge (Intermediate) or Can Use Power Bombs
```

Comment:
I think this can be safely lowered to beginner unless you have any arguments.

Reply:
I removed the trick gating since it's _technically_ vanilla, and what I had would have caused the pickup to be locked out with trickless door lock/entrance rando